### PR TITLE
Adding appveyor file to the repo with two stable builds

### PR DESCRIPTION
**Pull request by bsipocz**

### CODE REVIEW

**Comment by bsipocz** [comment]
@astrofrog - I've added aplpy/aplpy to the astropy projects (to avoid the inconvenience to log in/out). I assumed that the traffic here is rather low, so it won't add much bottleneck.

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+environment:
+
+  global:
+      PYTHON: "C:\\conda"
+      MINICONDA_VERSION: "latest"
+      CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci-helpers\\appveyor\\windows_sdk.cmd"
+      PYTHON_ARCH: "64" # needs to be set for CMD_IN_ENV to succeed. If a mix
+                        # of 32 bit and 64 bit builds are needed, move this
+                        # to the matrix section.
+      CONDA_DEPENDENCIES: "matplotlib pillow"
```
Comment (astrofrog) [inline appveyor.yml]: Need to add ``PIP_DEPENDENCIES: "pytest-mpl"``

Comment (bsipocz) [inline appveyor.yml]: Indeed. And what a nice bunch of new features GH has. :)

Comment (bsipocz) [inline appveyor.yml]: changes were made, but GH didn't recognized. Do you know a way to mark this as solved?

Comment (astrofrog) [inline appveyor.yml]: No automated way - I have to manually approve

**Comment by astrofrog** [comment]
Thanks!

**Pull request merged**